src/main.rs

```diff
@@ -573,6 +622,13 @@ impl AsyncComponent for App {
                 *self.exit_status_code.lock() = AppStatusCode::Restart;
                 relm4::main_application().quit();
             }
+            AppInput::ShouWindow => {
+                root.show();
+                root.maximize();
```

Comment:
Why is `.maximize()` needed?

Reply:
when windows is minimize, click open but do nothing.  just want to bring window to front,   hmm but this feature not work for linux,  maximize do nothing in this scenario.  do not thing it should be remove?

Reply:
Maximize makes window occupy the whole screen, we don't want that, in fact our window is not resizable in the first place.

What you probably wanted to do is `.unminimize()`, but even then it only makes sense if we minimized it first, which we didn't we hid the window. So I don't understand why this is necessary. If you want to print it to the front, maybe `present` is what should be used instead (it will also show and un-minimize the window for you).